sensing/autoware_cuda_pointcloud_preprocessor/src/cuda_pointcloud_preprocessor/outlier_kernels.cu

```diff
@@ -31,10 +30,10 @@ __global__ void ringOutlierFilterKernel(
     return;
   }
 
-  int min_i = max(i - num_points_threshold, 0);
-  int max_i = min(i + num_points_threshold, max_points_per_ring);
+  const int window_size = 5;
```

Comment:
This limits the maximum length of a checked walk to `5 + 1 + 5 = 11` points. While this did not cause any issues during review, this implicitly puts a limit on how big the object length computed between left/right point can get, possibly causing walks to be lost because the 11 points of space were still smaller than `object_length_threshold`.

For now, I think this is an acceptable workaround to bound computation time per point.

@amadeuszsz Opinions?

Reply:
@amadeuszsz Fully agree. For now however, let's prioritize this getting merged so people don't need to refactor all their launchers to use CUDA.